common/src/main/java/com/microsoft/identity/common/internal/net/HttpResponse.java

```diff
@@ -22,85 +22,35 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.internal.net;
 
-import androidx.annotation.NonNull;
-import androidx.annotation.Nullable;
-
-import net.jcip.annotations.Immutable;
-
 import java.util.Date;
 import java.util.List;
 import java.util.Map;
 
+import lombok.NonNull;
+
 /**
- * Internal class to wrap the raw server response, headers and status code.
+ * Deprecated
```

Comment:
Deprecated annotation?

Reply:
Intentionally not putting it, as @iambmelt called out earlier that it (or any warning) might break OneAuth.